# Point hessian is the internal structure of a map point


# the last two residuals
PointFrameResidualData[2] residuals
uint32[2] res_state